libminifi/test/TestBase.cpp

```diff
@@ -168,6 +168,30 @@ std::shared_ptr<core::Processor> TestPlan::addProcessor(const std::string &proce
   return addProcessor(processor_name, uuid, name, relationships, linkToPrevious);
 }
 
+std::shared_ptr<minifi::Connection> TestPlan::addConnection(const std::shared_ptr<core::Processor> source_proc, const core::Relationship& source_relationship, const std::shared_ptr<core::Processor>& destination_proc) {
+  std::stringstream connection_name;
+  connection_name << source_proc->getUUIDStr() << "-to-" << destination_proc->getUUIDStr();
```

Comment:
info: EFM names connections as `<source proc name>/<relationship name>/<dest proc name>`

Reply:
`TestPlan` does it like this. I would like to refactor this class at some point anyway, want me to change the names then?

Reply:
nope, it was just an info, since `TestPlan` does it like this and it is it's method, I'm good